src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -318,6 +373,63 @@ private void verifyFileContentForSession1InCourse1WithinSection1(String fileCont
+                "Question 2,\"Rate 1 other student's product\"",
+                "",
+                "Team,Giver's Full Name,Giver's Last Name,Giver's Email,Recipient's Team,Recipient's Full Name,Recipient's Last Name,Recipient's Email,Feedback",
+                "\"Team 1.1</td></div>'\"\"\",\"student1 In Course1</td></div>'\"\"\",\"Course1</td></div>'\"\"\",\"student1InCourse1@gmail.tmt\",\"Team 1.1</td></div>'\"\"\",\"student2 In Course1\",\"Course1\",\"student2InCourse1@gmail.tmt\",\"Response from student 1 to student 2.\"",
+                "\"Team 1.1</td></div>'\"\"\",\"student2 In Course1\",\"Course1\",\"student2InCourse1@gmail.tmt\",\"Team 1.1</td></div>'\"\"\",\"student1 In Course1</td></div>'\"\"\",\"Course1</td></div>'\"\"\",\"student1InCourse1@gmail.tmt\",\"Response from student 2 to student 1.\"",
+                "\"Team 1.1</td></div>'\"\"\",\"student3 In Course1\",\"Course1\",\"student3InCourse1@gmail.tmt\",\"Team 1.1</td></div>'\"\"\",\"student2 In Course1\",\"Course1\",\"student2InCourse1@gmail.tmt\",\"Response from student 3 \"\"to\"\" student 2. Multiline test.\"",
+                "",
+                "",
+                ""
+                // CHECKSTYLE.ON:LineLength
+        };
+
+        assertTrue(fileContent.startsWith(StringUtils.join(expected, Const.EOL)));
+    }
+
+    private void verifyFileContentForQuestion1Session1InCourse1WithinSection1(String fileContent,
+                              FeedbackSessionAttributes session) {
+        /*
+        full testing of file content is
+        in FeedbackSessionsLogicTest.testGetFeedbackSessionResultsSummaryAsCsv()
+        */
+
+        String[] expected = {
+                // CHECKSTYLE.OFF:LineLength csv lines can exceed character limit
+                "Course,\"" + session.getCourseId() + "\"",
+                "Session Name,\"" + session.getFeedbackSessionName() + "\"",
+                "Section Name,\"Section 1\"",
+                "",
+                "",
+                "Question 1,\"What is the best selling point of your product?\"",
+                "",
+                "Team,Giver's Full Name,Giver's Last Name,Giver's Email,Recipient's Team,Recipient's Full Name,Recipient's Last Name,Recipient's Email,Feedback",
+                "\"Team 1.1</td></div>'\"\"\",\"student1 In Course1</td></div>'\"\"\",\"Course1</td></div>'\"\"\",\"student1InCourse1@gmail.tmt\",\"Team 1.1</td></div>'\"\"\",\"student1 In Course1</td></div>'\"\"\",\"Course1</td></div>'\"\"\",\"student1InCourse1@gmail.tmt\",\"Student 1 self feedback.\"",
+                "\"Team 1.1</td></div>'\"\"\",\"student2 In Course1\",\"Course1\",\"student2InCourse1@gmail.tmt\",\"Team 1.1</td></div>'\"\"\",\"student2 In Course1\",\"Course1\",\"student2InCourse1@gmail.tmt\",\"I'm cool'\"",
+                "",
+                "",
+                ""
+                // CHECKSTYLE.ON:LineLength
+        };
+
+        assertTrue(fileContent.equals(StringUtils.join(expected, Const.EOL)));
+
+    }
```

Comment:
@wkurniawan07 

1. Moving the expected csv string to an external file is not really related to duplication. Point taken about readability. On the flip side, the string-encoded csv text is harder to eyeball for errors compared to an actual csv file one can open as a spreadsheet. But I'm ok to keep it as string in the code.

2. I asked to refactor because the same comment is repeated in many places, indicating a deeper level of duplication. The problem is we have a bunch of similar methods each serving only one call site. If the method takes both _expected_ string as well as _actual_ string as parameters (the current one takes only the actual string), all these similar methods can be folded into one.